slcp/process.py

```diff
@@ -72,12 +80,14 @@ def select_destination(self):
         If the destination path in arguments does not exist create it.
         :return: str. Destination folder path.
         """
+        root = tk.Tk()
+        root.withdraw()
         if self.args.dstcwd:
             destination = os.getcwd()
         else:
             if self.args.dest is None:
                 print("Choose a destination path.")
-                destination = os.path.normpath(askdirectory())
+                destination = os.path.normpath(askdirectory(title="slcp - Select the destination"))
```

Comment:
Is `root.destroy()` needed after we got a path from file dialog?

Reply:
Is it? As the widget was properly closed after the directory choice, I did not realize it.

Reply:
If I understood correctly the widget just go invisible but main loop of tk app is still going and tcl/tk interpreter stays running as well.

Reply:
Yes, that definitely makes sense.